### ISSUE TITLE
maint: set config value for ManyUUID limit

### ISSUE THREAD

**Issue by Forfold** [target/goalert#3381]
Most of our calls to `validate.ManyUUID` use a "magic number" of either 50, 100, or 200. It might be good to start a conversation around standardizing these values as global config variable:

<img width="453" alt="Screenshot 2023-10-27 at 9 52 43 AM" src="https://github.com/target/goalert/assets/11381794/42fccc19-4a9b-40f0-8b2e-d69963d675d8">

Reference: https://github.com/target/goalert/pull/3231#discussion_r1373493755

**Comment by mastercactapus** [comment]
Yeah, this is a good idea. At the very least, the reason the values are what they are can be documented easily that way.

I agree with @ethan-haynes on following the `search` package pattern; constant/fixed values will be easier to convey for API use since they are essential for limiting payload size/DB load/memory usage for a single request (and to keep our guarantees around things like response time).

Configurable limits are for things like data size-at-rest where you want to limit things like open alerts so the DB doesn't grow until it tips over if something goes wrong, but some installations may require them to be increased or decreased.

**Comment by Forfold** [comment]
By configurable, I basically meant the same as what Ethan proposed, so it works out 🙂

**Comment by mastercactapus** [comment]
Maybe part of the validation package?

That way the code could read like:
```go
validate.ManyUUID("Omit", ids, validate.MaxOmit)
```

Alternatively, we could look into moving from UUID-strings to actual UUIDs (move the parsing to the generated GraphQL code) -- then we could move the length-limit to the schema via directive potentially, which could look like this:

```graphql
input ServiceSearchOptions {
  first: Int = 15
  after: String = ""
  search: String = ""
  omit: [ID!] @max(length: 50)
```

The latter I don't think could use a constant value, but it would be all in one place, discoverable, and scannable (i.e., you could search for `omit:` and validate they all have the same value).
